pineappl_cli/src/evolve.rs

```diff
@@ -37,28 +32,124 @@ fn evolve_grid(
         targetpids: Vec<i32>,
     }
 
-    let mut archive = Archive::new(File::open(eko)?);
+    #[derive(Default, Deserialize)]
+    struct Rotations {
+        #[serde(rename = "_targetgrid")]
```

Comment:
`_targetgrid` is saved without a prepended underscore.

Essentially, the conversion you're doing here is already done while saving.

Reply:
Unfortunately, this has recently changed...

Reply:
In the `metadata.yaml` file that I'm looking at the format is:

```yaml
rotations:
  _targetgrid:
```

Reply:
Ok, then you got an old EKO...

https://github.com/NNPDF/eko/blob/9538753139cfcbafc81f1851449deab4fe59f9b7/src/eko/io/runcards.py#L206-L209

Reply:
Oh I see, but no worries, as you can see `serde` allows to very easily support different formats.

Reply:
Commit 5378a41d020c6db9f16030bc0b4fe60d41b1de31 should support both (as the line you pointed out in EKO). I continue to be impressed by serde!